website/templates/project/project.mako

```diff
@@ -188,10 +188,12 @@
                 <span data-bind="css: icon"></span>
                 </p>
 
-                % if node['description'] or 'write' in user['permissions']:
-                    <p>
+                % if not node['description']:
+                    <p></p>
```

Comment:
This extra `p` tag will result in unwanted whitespace on the page. Can you simplify your logic so that this can be removed?